src/SGFTree.cpp

```diff
@@ -268,6 +282,8 @@ void SGFTree::populate_states() {
 void SGFTree::copy_state(const SGFTree& tree) {
     m_initialized = tree.m_initialized;
     m_state = tree.m_state;
+    m_loaded_timecontrol = tree.m_loaded_timecontrol;
```

Comment:
Are there cases in which it's necessary to copy this?

Reply:
I wasn’t sure so I put it in..

Reply:
I wasn’t sure so I put it in..